runbotics/runbotics-orchestrator-ui/src/src-app/components/ProcessVariablesPanel/ProcessVariablesTabs.tsx

```diff
@@ -0,0 +1,38 @@
+
+import React, {useState} from 'react';
+
+import TabContext from '@mui/lab/TabContext';
+import TabList from '@mui/lab/TabList';
+import TabPanel from '@mui/lab/TabPanel';
+import { Box, Tab } from '@mui/material';
+
+
+const ProcessVariablesTabs = ({tabs}) => {
```

Comment:
type this component, (eg: ProcessVariablesTabs: FC<ProcessVariablesTabsProps>) where ProcessVariablesTabs is in the shape of the props

Reply:
this component will not be used at the end, but noted for the future